app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -492,14 +498,19 @@ class StateFragmentTest {
 
   @Test
   fun testStateFragment_loadExp_secondState_submitWrongAnswer_contentDescriptionIsCorrect() {
-    launchForExploration(TEST_EXPLORATION_ID_2, shouldSavePartialProgress = false).use {
+    launchForExploration(TEST_EXPLORATION_ID_2, shouldSavePartialProgress = false).use { scenario ->
       startPlayingExploration()
       clickContinueInteractionButton()
 
       // Attempt to submit an wrong answer.
       typeFractionText("1/4")
       clickSubmitAnswerButton()
 
+      scenario.onActivity { activity ->
+        val view = activity.findViewById<android.view.View>(R.id.submitted_answer_text_view)
+        println(view.contentDescription)
```

Comment:
Are we keeping this or it was just for debugging?

Reply:
Nice catch. Removed.